adafruit_bus_device/i2c_device.py

```diff
@@ -60,7 +63,7 @@ class I2CDevice:
                 device.write(bytes_read)
     """
 
-    def __init__(self, i2c, device_address, probe=True, timeout=0.25):
+    def __init__(self, i2c, device_address, probe=True, timeout=250):
```

Comment:
Please go back to floating point seconds. We are trying to be uniform and always use float seconds.